scripts/helpers/utils.mts

```diff
@@ -119,6 +119,22 @@ export function partitionArguments(
     : [args, []];
 }
 
+export function partitionArgumentsV2(
```

Comment:
(optional) nit: is there a more descriptive name we could use here instead of `V2`? Or an overload since it seems that the only "addition" is the logic for the `defaultArgs` parameter.

Reply:
100% agreed. `partitionArgumentsV2` without the third argument is identical to `partitionArguments` so it seems like they are definitely mergeable.

Reply:
Actually now that the scripts are unified, I think the "V2" versions of the helpers I wrote can just usurp the originals.